### PR TITLE
Add some missing features

### PR DESCRIPTION
**Pull request by jeffswartz**
Adds API enhancements for features missing from the OpenTok iOS and Android SDKs.

### Contributing checklist
- [√] Code must follow existing styling conventions
- [√] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# 0.26.0 (August 2023)
+
+- [Update]: Add API to implement functionality missing from the OpenTok Andriod and iOS SDKs:
```
Comment (v-kpheng) [inline CHANGELOG.md:3]: Nit: misspelled Android

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java [hunk 2]
```diff
@@ -290,6 +311,59 @@ public void disconnectSession(String sessionId, Callback callback) {
         }
     }
 
+    @ReactMethod
+    public void forceMuteAll(String sessionId, ReadableArray excluededStreamIds, Promise promise) {
```
Comment (v-kpheng) [inline android/src/main/java/com/opentokreactnative/OTSessionManager.java:315]: Nit: misspelled "excluded"

File: CHANGELOG.md (hunk 1 above)
Comment (jeffswartz) [inline CHANGELOG.md:3]:
```suggestion
- [Update]: Add API to implement functionality missing from the OpenTok Android and iOS SDKs:
```

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java [hunk 3]
```diff
@@ -290,6 +311,59 @@ public void disconnectSession(String sessionId, Callback callback) {
         }
     }
 
+    @ReactMethod
+    public void forceMuteAll(String sessionId, ReadableArray excludedStreamIds, Promise promise) {
+        ConcurrentHashMap<String, Session> mSessions = sharedState.getSessions();
+        Session mSession = mSessions.get(sessionId);
+        ConcurrentHashMap<String, Stream> streams = sharedState.getSubscriberStreams();
+        ArrayList<Stream> mExcluededStreams = new ArrayList<Stream>();
```
Comment (v-kpheng) [inline android/src/main/java/com/opentokreactnative/OTSessionManager.java:319]: Nit: `mExcluededStreams` -> `mExcludedStreams`.  Perhaps other examples elsewhere too.

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java [hunk 4]
```diff
@@ -492,6 +604,33 @@ public void getSessionInfo(String sessionId, Callback callback) {
...
+    @ReactMethod
+    public void getSessionCapabilities(String sessionId, Callback callback) {
+        ConcurrentHashMap<String, Session> mSessions = sharedState.getSessions();
+        Session mSession = mSessions.get(sessionId);
+        WritableMap sessionCapabilitiesMap = Arguments.createMap();
+        if (mSession != null){
+            Session.Capabilities sessionCapabilities = mSession.getCapabilities();
+            sessionCapabilitiesMap.putBoolean("canForceMute", sessionCapabilities.canForceMute);
+            sessionCapabilitiesMap.putBoolean("canPublish", sessionCapabilities.canPublish);
+            // Bug in OT Android SDK. This should always be true, but it is set to false:
```
Comment (v-kpheng) [inline android/src/main/java/com/opentokreactnative/OTSessionManager.java:616]: Excellent comment! 💪

File: docs/EventData.md [hunk 5]
```diff
@@ -114,11 +114,72 @@ You can find the structure of the object below:
...
+```
+
+## PublisherVideoNetworkStatsEvent
+
+To get video data for a publisher, register an event listener for the OTPublisher
+`videoNetworkStats` event. The object has the following structure: 
+
+```javascript
+  event = [
+      {
```
Comment (v-kpheng) [inline docs/EventData.md:138]: Nit: formatting is off.

File: docs/EventData.md [hunk 6]
```diff
@@ -114,11 +114,72 @@ You can find the structure of the object below:
...
+the OTPublisher `videoNetworkStats` event.
+
+## PublisherAudioNetworkStatsEvent
+
+To get audio data for a publisher, register an event listener for the OTPublisher
+`audioNetworkStats` event. The object has the following structure: 
+
+```javascript
+  event = [
+      {
```
Comment (v-kpheng) [inline docs/EventData.md:159]: Same nit about tidying here -- likely more examples elsewhere too.

File: docs/EventData.md [hunk 7]
```diff
@@ -263,3 +324,37 @@ You can find the structure of the object below:
...
+    },
+    jsonArrayOfReports: string
+  };
+```
+
+## PublisherRtcStatsReportEvent
+
+```javascript
+  event = [
+    connectionId: string
```
Comment (v-kpheng) [inline docs/EventData.md:357]: Nit: missing ending comma.

File: docs/OTSession.md [hunk 8]
```diff
@@ -172,6 +178,37 @@ feature. See the [OpenTok IP Proxy](https://tokbox.com/developer/guides/ip-proxy
...
+
+**reportIssue()** Lets you report that your app experienced an issue (to view with
+[Inspector](http://tokbox.com/developer/tools/Inspector) or to discuss with the Vonage API
+support team.) The method returns a Promise that resolves with a string, the issue ID.
+
+**forceMuteAll()** Forces all publishers in the session (except for those publishing excluded streams) to mute audio. 
+
+This method has one optional parameter -- `excludedStreams`, and array of stream IDs. A stream published by the moderator calling the forceMuteAll() method is muted along with other streams in the session, unless you add the moderator's stream (or streams) to the `excludedStreams` array. If you leave out the `excludedStreams` parameter, all streams in the session (including those of the moderator) will stop publishing audio. Also, any streams that are published after the call to the `forceMuteAll()` method are published with audio muted. You can remove the mute state of a session by calling the `OTSession.disableForceMute()` method.
+
+After you call the Session.disableForceMute() method, new streams published to the session will no longer have audio muted.
```
Comment (v-kpheng) [inline docs/OTSession.md:202]: Nit: enclose this mute method with ticks?

File: ios/OpenTokReactNative/OTSessionManager.swift [hunk 9]
```diff
@@ -317,6 +341,73 @@ class OTSessionManager: RCTEventEmitter {
         callback([sessionInfo]);
     }
     
+    @objc func getSessionCapabilities(_ sessionId: String, callback: RCTResponseSenderBlock) -> Void{
+        guard let session = OTRN.sharedState.sessions[sessionId] else { callback([NSNull()]); return }
+        var sessionCapabilities: Dictionary<String, Any> = [:];
+        sessionCapabilities["canPublish"] = session.capabilities?.canPublish;
```
Comment (v-kpheng) [inline ios/OpenTokReactNative/OTSessionManager.swift:347]: Is this still needed, given the bug (and code) below?

File: ios/OpenTokReactNative/Utils/EventUtils.swift [hunk 10]
```diff
@@ -55,6 +55,47 @@ class EventUtils {
...
+
+    static func preparePublisherAudioNetworkStats(_ audioStats: [OTPublisherKitAudioNetworkStats]) -> [Dictionary<String, Any>] {
+        var statsArray:[Dictionary<String, Any>] = [];
+        for value in audioStats {
+            var stats: Dictionary<String, Any> = [:];
+            stats["connectionId"] = value.connectionId;
+            stats["subscriberId"] = value.subscriberId;
+            stats["audioPacketsLost"] = value.audioPacketsLost;
+            stats["audioPacketsSent"] = value.audioPacketsSent;
+            stats["audioPacketsSent"] = value.audioPacketsSent;
```
Comment (v-kpheng) [inline ios/OpenTokReactNative/Utils/EventUtils.swift:93]: Nit: this is being set twice.

Comment (v-kpheng) [reply ios/OpenTokReactNative/Utils/EventUtils.swift:93]: Perhaps you meant "audioBytesSent"?

File: src/OTSession.js [hunk 11]
```diff
@@ -93,10 +93,41 @@ export default class OTSession extends Component {
...
+          reject (new Error(error))
+        }
+      });
+    })
+  }
   signal(signal) {
     const signalData = sanitizeSignalData(signal);
     OT.sendSignal(this.props.sessionId, signalData.signal, signalData.errorHandler);
   }
+  forceMuteAll(excludedStreamIds) {
```
Comment (v-kpheng) [inline src/OTSession.js:122]: Nit and IMO: more idiomatic would be `forceMuteAll(excludedStreamIds = [])`.

**Review by v-kpheng — APPROVED** [review]
LGTM! 💪 🚀 

That said, an audio stat may be missing (see review).  In any case, we can fix in a subsequent PR if needed.

**Pull request merged**